src/components/StatusBadge/StatusBadge.stories.tsx

```diff
@@ -62,3 +63,23 @@ export const LowerBarrier: Story = {
     </VStack>
   ),
 };
+
+export const Icons: Story = {
+  render: () => (
+    <VStack spacing="s" align="stretch">
+      <StatusBadge level="high">
+        <Icon color="brand.primary" mr="2" name="errorFilled" size="medium" />
+        On-Site Access Only
+      </StatusBadge>
+      <StatusBadge level="low">
+        <Icon
+          color="ui.black"
+          mr="2"
```

Comment:
Right. Let's stick to being consistent and let the dev/consuming app add the appropriate styles in the `Icon` when it is passed. This means we may need more code snippets if the icon is added at the end rather than at the beginning.

Do add that we highly recommend the styles when adding the example and that the "xs" value is consistent with NYPL designs.

Reply:
Okay! Take a look at what I pushed up and lmk if that is aligned with what you are thinking.